src/users/store/effects/users.effects.ts

```diff
@@ -132,4 +132,16 @@ export class UsersEffects {
       map(() => new fromRoot.Go({ path: ['users/invite-user'] }))
     )
   );
+
+  public getAccessTypes$ = createEffect(() =>
+  this.actions$.pipe(
+    ofType(usersActions.LOAD_ACCESS_TYPES),
+    switchMap((action: any) => {
+      return this.usersService.retrieveAccessType(action.payload).pipe(
```

Comment:
The access types returned by the API operation makes me think this belongs in the organisation store rather than the user store. This should avoid mixing up the user access types for a user with organisation access types